docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -466,6 +466,16 @@ So if you want to deploy your application somewhere, you need to copy the _runne
 
 NOTE: Before running the application, don't forget to stop the hot reload mode (hit `CTRL+C`), or you will have a port conflict.
 
+=== Using fast-jar
+
+The previous section assumes that the configuration property `quarkus.package.type` has either not been set or has been set to `legacy-jar`. When the property has been set to
+`fast-jar` (which will become the default soon), then the result of executing `./mvnw package` is a new directory under `target` named `quarkus-app`.
```

Comment:
The gradle tooling doc addition gives the full specific example of setting `quarkus.package.type=fast-jar`, and notes it can be set in `application.properties` (or other supported sources). Although the overall idea is certainly covered here as-is, this text in comparison lacks the full example or any specific indication where it can be set. I think both would be more helpful to someone who is truly only getting started, so I think it might be clearer if this bit used an approach more similar to the gradle segment text, perhaps keeping the intro sentence about being unset/legacy-jar.

Reply:
I made a small change based on your feedback.

Thanks